images/discord.svg

```diff
@@ -0,0 +1,18 @@
+<path d="M89.8227 17.9433C88.8383 17.4551 88.0878 16.7877 87.5713 15.9383C87.0547 15.089 86.794 14.1426 86.794 13.0965C86.794 12.0504 87.0523 11.1164 87.5713 10.297C88.0903 9.47756 88.8359 8.83497 89.8154 8.36922C90.7925 7.90347 91.9596 7.67183 93.3119 7.67183C94.6642 7.67183 95.8313 7.90347 96.8084 8.36922C97.7855 8.83497 98.5311 9.47258 99.0428 10.287C99.5544 11.1015 99.8103 12.0355 99.8103 13.094C99.8103 14.1401 99.5544 15.0865 99.0428 15.9358C98.5311 16.7852 97.783 17.4527 96.7987 17.9408C95.8143 18.429 94.652 18.6731 93.3095 18.6731C91.9669 18.6731 90.8046 18.4315 89.8227 17.9433ZM95.0053 14.8499C95.4195 14.4265 95.6291 13.8661 95.6291 13.1687C95.6291 12.4713 95.422 11.9159 95.0053 11.505C94.5911 11.0915 94.0258 10.8848 93.3095 10.8848C92.5809 10.8848 92.0108 11.0915 91.5941 11.505C91.1799 11.9184 90.9728 12.4713 90.9728 13.1687C90.9728 13.8661 91.1799 14.4265 91.5941 14.8499C92.0083 15.2733 92.5809 15.4875 93.3095 15.4875C94.0258 15.485 94.5911 15.2733 95.0053 14.8499Z" fill="#5865F2"/>
+<path d="M110.059 8.31932V12.0852C109.625 11.7938 109.065 11.6493 108.37 11.6493C107.461 11.6493 106.76 11.9308 106.272 12.4937C105.782 13.0566 105.539 13.9333 105.539 15.1188V18.3268H101.36V8.12754H105.454V11.3704C105.68 10.1848 106.048 9.3106 106.555 8.74522C107.059 8.18233 107.712 7.90089 108.507 7.90089C109.108 7.90089 109.625 8.04037 110.059 8.31932Z" fill="#5865F2"/>
+<path d="M123.942 5.77394V18.3269H119.763V16.0429C119.41 16.9022 118.874 17.5572 118.153 18.0056C117.432 18.4514 116.54 18.6756 115.482 18.6756C114.537 18.6756 113.713 18.4414 113.009 17.9707C112.305 17.5 111.762 16.8549 111.379 16.0355C110.999 15.216 110.806 14.2895 110.806 13.2534C110.794 12.1849 110.997 11.226 111.413 10.3767C111.827 9.52736 112.415 8.86484 113.17 8.38913C113.925 7.91341 114.788 7.67431 115.755 7.67431C117.746 7.67431 119.081 8.55849 119.763 10.3244V5.77394H123.942ZM119.14 14.7802C119.566 14.3567 119.778 13.8063 119.778 13.1338C119.778 12.4838 119.571 11.9533 119.157 11.5473C118.742 11.1413 118.175 10.9371 117.461 10.9371C116.757 10.9371 116.194 11.1438 115.772 11.5572C115.351 11.9707 115.141 12.5062 115.141 13.1687C115.141 13.8312 115.351 14.3717 115.772 14.7901C116.194 15.2086 116.749 15.4178 117.444 15.4178C118.148 15.4153 118.713 15.2036 119.14 14.7802Z" fill="#5865F2"/>
+<path d="M59.1895 9.21341C60.3401 9.21341 61.2728 8.35701 61.2728 7.30058C61.2728 6.24416 60.3401 5.38775 59.1895 5.38775C58.039 5.38775 57.1063 6.24416 57.1063 7.30058C57.1063 8.35701 58.039 9.21341 59.1895 9.21341Z" fill="#5865F2"/>
+<path d="M57.1044 10.531C58.3812 11.0939 59.9649 11.1188 61.271 10.531V18.3816H57.1044V10.531Z" fill="#5865F2"/>
+</g>
+<defs>
+<clipPath id="clip0">
+<rect width="124" height="24" fill="white"/>
+</clipPath>
+</defs>
+</svg>
```

Comment:
Should we be hosting this?  What if it changes and there are legal ramifications?

Reply:
Changed to a direct link